specs/phase0/validator.md

```diff
@@ -102,10 +104,17 @@ We define the following Python custom types for type hinting and readability:
 | `EPOCHS_PER_SUBNET_SUBSCRIPTION` | `2**8` (= 256) | epochs | ~27 hours |
 | `ATTESTATION_SUBNET_COUNT` | `64` | The number of attestation subnets used in the gossipsub protocol. |
 | `ATTESTATION_SUBNET_EXTRA_BITS` | `0` | The number of extra bits of a NodeId to use when mapping to a subscribed subnet |
-| `SUBNETS_PER_NODE` | `2` | The number of long-lived subnets a beacon node should be subscribed to. |
 | `ATTESTATION_SUBNET_PREFIX_BITS` | `(ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS)` | |
 | `NODE_ID_BITS` | `256` | The bit length of uint256 is 256 |
 
+## Configuration
+
+### Misc
+
+| Name | Value | Unit | Duration |
+| - | - | :-: | :-: |
+| `SUBNETS_PER_NODE` | `2` | The number of long-lived subnets a beacon node should be subscribed to. |
```

Comment:
I'm not sure this should go into validator.md. As it pertains to *all* nodes rather than just validator nodes

maybe in the p2p spec? what do you think

Reply:
Agreed. Hmm, I can move the whole "## Phase 0 attestation subnet stability" section to p2p spec.